### PR TITLE
Refactor fuse low level implementation

### PR DESCRIPTION
**Pull request by cyshi**
Refactor fuse low level implementation

### CODE REVIEW

File: fuse_lowlevel/bfs_ll_mount.cc [hunk 1]
```diff
@@ -1,87 +1,105 @@
...
-static void bfs_ll_link(fuse_req_t req, fuse_ino_t ino, fuse_ino_t tparent, const char *tname){
+static void bfs_ll_link(fuse_req_t req, fuse_ino_t ino, fuse_ino_t tparent, const char *tname) {
     fprintf(stderr, BFS"%s(target name:%s)\n", __func__, tname);
 }
 
-static void bfs_ll_unlink(fuse_req_t req, fuse_ino_t parent, const char *name){
+static void bfs_ll_unlink(fuse_req_t req, fuse_ino_t parent, const char *name) {
     fprintf(stderr, BFS"%s(%s)\n", __func__, name);
-}
+ }
```
Comment (lylei) [inline fuse_lowlevel/bfs_ll_mount.cc]: A redundant space？

File: fuse_lowlevel/bfs_ll_mount.cc [hunk 2]
```diff
@@ -1,87 +1,105 @@
...
+static void bfs_ll_rmdir(fuse_req_t req, fuse_ino_t parent, const char *name) {
     fprintf(stderr, BFS"%s(%s)\n", __func__, name);
 }
 
-static void bfs_ll_fsync(fuse_req_t req, fuse_ino_t ino, int datasync, struct fuse_file_info *fi){
+static void bfs_ll_fsync(fuse_req_t req, fuse_ino_t ino, 
+                         int datasync, struct fuse_file_info *fi) {
     fprintf(stderr, BFS"%s\n", __func__);
-}
+ }
```
Comment (lylei) [inline fuse_lowlevel/bfs_ll_mount.cc]: A redundant space？

File: fuse_lowlevel/bfs_ll_mount.cc [hunk 3]
```diff
@@ -123,55 +141,68 @@ int parse_bfs_args(int* argc, char* argv[]) {
...
+        .rename = bfs_ll_rename;
+        .link = bfs_ll_link;
+        .unlink = bfs_ll_unlink;
+        .rmdir = bfs_ll_rmdir;
+        .fsync = bfs_ll_fsync;
+        .fsyncdir = bfs_ll_fsyncdir;
+        .access = bfs_ll_access;
+    };
+
+    if(parse_bfs_args(&argc, argv) != 0) {
```
Comment (lylei) [inline fuse_lowlevel/bfs_ll_mount.cc]: need a space after `if`

File: fuse_lowlevel/bfs_ll_mount.cc [hunk 4]
```diff
@@ -1,87 +1,105 @@
-#include <fuse_lowlevel.h>
```
Comment (yvxiang) [inline fuse_lowlevel/bfs_ll_mount.cc]: copyright?

File: fuse_lowlevel/bfs_ll_mount.cc (hunk 1 above)
Comment (cyshi) [inline fuse_lowlevel/bfs_ll_mount.cc]: fixed

File: fuse_lowlevel/bfs_ll_mount.cc (hunk 2 above)
Comment (cyshi) [inline fuse_lowlevel/bfs_ll_mount.cc]: fixed

File: fuse_lowlevel/bfs_ll_mount.cc (hunk 3 above)
Comment (cyshi) [inline fuse_lowlevel/bfs_ll_mount.cc]: fixed

File: fuse_lowlevel/bfs_ll_mount.cc (hunk 4 above)
Comment (yvxiang) [inline fuse_lowlevel/bfs_ll_mount.cc]: 嗯，我是说是不是需要另外加个版权上去

Comment (cyshi) [inline fuse_lowlevel/bfs_ll_mount.cc]: 已加

**Pull request merged**